### PR TITLE
Gitignore

### PR DESCRIPTION
**Pull request by joaherrerama**
Adding .gitignore file

### CODE REVIEW

File: actinia_stac_plugin/api/stac.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def delete(self):
 
         return make_response(deleted_stac,200)
 
-class StacCollecions(ResourceBase):
+class StacCollections(ResourceBase):
```
Comment (anikaweinmann) [inline actinia_stac_plugin/api/stac.py:68]: Do the and the following changes belong in this pull request?

File: actinia_stac_plugin/api/stac.py [hunk 2]
```diff
@@ -83,3 +83,12 @@ def get(self, collection_id):
...
 
+class StacCollectionList(ResourceBase):
+
+    def get(self): 
+        return ""
+
+class StacInstanceList(ResourceBase):
+
+    def get(self): 
+        return ""
```
Comment (anikaweinmann) [inline actinia_stac_plugin/api/stac.py:94]: I think no new line at the end of a file is not the standard. Maybe you want to include a linting directly e.g. like in https://github.com/mundialis/actinia-module-plugin/blob/main/.github/workflows/flake8.yml. With exceptions as in https://github.com/mundialis/actinia-module-plugin/blob/main/.flake8, if needed.

**Comment by marcjansen** [comment]
I'd prefer to keep the changes separate, so 2 PRs: one with `.gitignore` and another with the rest, if you think that the second is ready.

Ping me if you need help with splitting the changes up.

File: actinia_stac_plugin/api/stac.py (hunk 2 above)
Comment (marcjansen) [reply actinia_stac_plugin/api/stac.py:94]: I think @mmacata wanted to work on this in #4

Comment (marcjansen) [reply actinia_stac_plugin/api/stac.py:94]: And yes, having a newline at the end of the file would be my preference

File: actinia_stac_plugin/api/stac.py (hunk 1 above)
Comment (joaherrerama) [reply actinia_stac_plugin/api/stac.py:68]: Not really, they are related to the commit right before, I would need to split the changes

**Review by marcjansen — APPROVED** [review]
Great thanks for this.

File: actinia_stac_plugin/api/stac.py (hunk 2 above)
Comment (mmacata) [reply actinia_stac_plugin/api/stac.py]: I talked to Jorge, he wants to learn it as well so he will look into it soon.

**Pull request merged**